<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SQLManager</class>
 <widget class="QDialog" name="SQLManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="username">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="continueButton">
       <property name="text">
        <string>Continue</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name1Edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name2Edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="rerollname">
       <property name="text">
        <string>Reroll</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
